README.rst

```diff
@@ -875,19 +875,59 @@ Get command details
 
     GET /api/v1/controller/device/{device_id}/command/{command_id}/
 
-Get device coordinates
+Get device location
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/controller/device/{id}/location/?key={device key}
+
+**Note**: Device Location endpoints can only be accessed with the
+Device ``key`` or User ``Token``.
+
+Update device location
 ^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: text
 
-    GET /api/v1/controller/device/{id}/location/
+    PUT /api/v1/controller/device/{id}/location/?key={device key}
```

Comment:
When the device doesn't have a location, there is no indication of error.
Which is okay, but here in the docs, I think we should indicate the fact that user needs to check if location is null on their end, else it will be missed edge case ! :smile:

What do you think?

Reply:
Okay I will update it in the docs.

Reply:
Done.